### DISCUSSION TITLE
FX/Reverb tails cut when muting channel

### DISCUSSION THREAD

**Discussion by randorec** [monocasual/giada#569] [General]
I'm new to Giada so perhaps I'm not doing things correctly. While playing through different variations of my song I mute channels and bring them back. I noticed that if I have an FX assigned, like a reverb or delay, the FX is muted immediately when the channel is muted. I expected that the sample would stop but the reverb tail would continue. This is what would happen if a musician stopped playing in a room or hall and what I expected. Is there a control for this in the software that I am missing or is this how the channels are working right now?

If it's the latter, I'd suggest that allowing the sound to be muted while letting the FX continue is more musical and the user should have the option.

**Comment by gvnnz** [comment]
Hey @randorec , welcome. Yes you are right, the mute button sits at the very end of the channel processing: it mutes everything that comes out of it. I think you raised a good point here: do you think the mute behavior should be configurable by the user? Is there a good scenario where the current mode would be actually useful?

**Reply by randorec** [reply]
Since a normal mixing desk would just affect the direct signal and not the effects loop I don't think there's a need for the channel to mute the effect too. But of course, as soon as I say that, someone will come up with a use case critical to their workflow. It might be more work to have a pre/post effect mute choice but that would probably satisfy everyone. However, I'd say that the default should be to mute pre-effect to let the tail of the effect continue.

**Comment by CTBlaze** [comment]
Well, there are FX that keep creating sound or noise, which should be possible to be muted.
Tails of delay(!) and reverb are also important, of course... 
so IMHO there should be a configurable option for EACH of the channel FX: "Link to MUTE".

**Comment by Simon-L** [comment]
It is fine for the mute button to mute the entire channel. Still the use case for keeping the effects running has to be satisfied. One very common concept for that are FX buses working with sends and returns.
Not sure if this has evolved since the last messages?

**Reply by gvnnz** [reply]
@Simon-L we are still collecting ideas at the moment. Good point for the FX buses, something to keep in mind for https://github.com/monocasual/giada/issues/286.
